packages/js/src/plugins/auctionHouseModule/operations/createBid.ts

```diff
@@ -88,22 +178,54 @@ export type CreateBidInput = {
  * @category Outputs
  */
 export type CreateBidOutput = {
-  /** The blockchain response from sending and confirming the transaction. */
-  response: SendAndConfirmTransactionResponse;
+  /** Buyer trade state account PDA encoding the bid order. */
   buyerTradeState: Pda;
+
+  /** The asset's token account address in case the bid is private. */
   tokenAccount: Option<PublicKey>;
+
+  /** The asset's metadata PDA. */
   metadata: Pda;
+
+  /** The potential buyer of the asset. */
   buyer: PublicKey;
+
+  /** The PDA of the receipt account in case it was printed. */
   receipt: Option<Pda>;
+
+  /** The address of the bookkeeper wallet responsible for the receipt. */
   bookkeeper: Option<PublicKey>;
+
+  /** The buyer's price. */
   price: SolAmount | SplTokenAmount;
+
+  /** The number of tokens to bid for. */
   tokens: SplTokenAmount;
+
+  /** A model that keeps information about the Bid. */
   bid: Bid;
+
+  /** The blockchain response from sending and confirming the transaction. */
+  response: SendAndConfirmTransactionResponse;
 };
 
 /**
- * @group Operations
- * @category Handlers
+ * Creates a bid on a given asset.
+ *
+ * You can post a public bid on a non-listed NFT by skipping seller and tokenAccount properties.
+ * Public bids are specific to the token itself and not to any specific auction.
+ * This means that a bid can stay active beyond the end of an auction
+ * and be resolved if it meets the criteria for subsequent auctions of that token.
+ *
+ * ```ts
+ * const transactionBuilder = metaplex
+ *   .auctionHouse()
+ *   .builders()
+ *   .createBid({ auctionHouse, mintAccount, seller });
+ * ```
+ *
+ * @group Transaction Builders
+ * @category Constructors
```

Comment:
I think this change is a typo and should be reverted. Operation Handlers do not need this comment, only Operations and Transaction Builders. (it's written 3 times in this file instead of twice).